edit-post/assets/stylesheets/_z-index.scss

```diff
@@ -29,6 +29,17 @@ $z-layers: (
 	'.edit-post-header': 30,
 	'.wp-block-image__resize-handlers': 1, // Resize handlers above sibling inserter
 
+	// Should have lower index than anything else positioned inside the block containers
+	'.editor-block-list__block-drag-inset': 0,
+	// Should have higher index than anything else positioned inside the block containers
+	'.editor-block-list__block-drag-inset.is-visible > .inner': 1000000000,
```

Comment:
These numbers are becoming increasingly... increasing. Why does this need to be 1000x higher than the next-highest z-index? The issue with z-index is we always think _our thing_ ought to be higher than _all the things_, but when all elements act this way, it's a never-ending battle.

Reply:
Agreed - one of those things that need to be cleaned up. I just bumped up those values to make sure they are working as expected. Not sure though what the best way to ensure they *"have higher index than anything else positioned inside the block containers"*.